src/GraphQL/Types/Composite/AutoRegisteringHelper.cs

```diff
@@ -69,5 +81,40 @@ private static string GetPropertyName<TSourceType>(Expression<Func<TSourceType,
 
             throw new NotSupportedException($"Unsupported type of expression: {expression.GetType().Name}");
         }
+
+        /// <summary>
+        /// Constructs a lambda expression for a field resolver to return the specified query argument
+        /// from the resolve context.
+        /// </summary>
+        internal static LambdaExpression GetParameterExpression(Type parameterType, QueryArgument queryArgument)
+        {
+            //construct a call to ResolveFieldContextExtensions.GetArgument, passing in the appropriate default value
```

Comment:
GetArgument or TryGetArgument?

Reply:
Needs to be updated to AutoRegisteringHelper.GetArgumentInternal and etc.

Reply:
XML comments have been updated.